test/src/test/java/org/corfudb/CorfuTestParameters.java

```diff
@@ -89,7 +89,7 @@ public CorfuTestParameters(){
         TIMEOUT_VERY_SHORT = Duration.of(500, MILLIS);
         TIMEOUT_SHORT = Duration.of(1, SECONDS);
         TIMEOUT_NORMAL = Duration.of(10, SECONDS);
-        TIMEOUT_LONG = Duration.of(2, MINUTES);
+        TIMEOUT_LONG = Duration.of(5, MINUTES);
```

Comment:
Can we keep the timeout duration for tests at two minutes? What is the need in bumping it so high?

Reply:
@Jas0nch had done it probably to avoid some tests failing, as this is used as the max allowed time for each test before it is marked as failed, but I think they will fail anyways even after increasing the time. I'll revert and try. Thanks.